### PR TITLE
PAYARA-189 Remove GMS and Shoal

### PR DESCRIPTION
**Pull request by smillidge**
Remove GMS/Shoal and replace with Hazelcast equivalent functionality

### CODE REVIEW

**Comment by smillidge** [comment]
jenkins quick test payara 5 please

**Comment by Pandrex247** [comment]
Jenkins, quick test Payara 5 please

File: appserver/orb/orb-iiop/src/main/java/org/glassfish/enterprise/iiop/impl/IiopFolbGmsClient.java [hunk 1]
```diff
@@ -39,6 +39,8 @@
  */
 // Portions Copyright [2017] [Payara Foundation and/or its affiliates]
 
+// Portions Copyright [2017] [Payara Foundation and/or its affiliates]
```
Comment (michaelranaldo) [inline appserver/orb/orb-iiop/src/main/java/org/glassfish/enterprise/iiop/impl/IiopFolbGmsClient.java]: Redundant license

**Comment by Pandrex247** [comment]
Continued in https://github.com/payara/Payara/pull/2211

**Pull request closed without merging**